### DISCUSSION TITLE
Customize Lists

### DISCUSSION THREAD

**Discussion by kaganergovan** [community/community#51782] [Other Feature Feedback, Questions, & Ideas]
When I tried Lists on Github mobile (for iPadOS) I realised that would be useful to categorize lists by customizing them with icons and colors like "Apple Reminders" lists. I attached an example below

![image](https://user-images.githubusercontent.com/8795682/229377566-e6cc6036-b45d-4cc0-b276-1802d39753ee.jpeg)

Sincerely to all dev friends,

**Comment by shaedrich** [comment]
One workaround now is to use emojies as icons

**Reply by Sambyah9595** [reply]
> - ****

**Reply by Sambyah9595** [reply]
{"Status":"Endpoint not found"}

**Reply by Sambyah9595** [reply]
No nzn
